src/langchain_google_memorystore_redis/vector_store.py

```diff
@@ -448,8 +465,9 @@ def from_texts(
         texts: List[str],
         embedding: Embeddings,
         metadatas: Optional[List[dict]] = None,
-        client=None,
-        index_name=None,
+        ids: Optional[List[str]] = None,
```

Comment:
nit: Optional[List[dict]] should be equivalent to List[dict] | None ([ref](https://stackoverflow.com/questions/69440494/python-3-10-optionaltype-or-type-none)). Either one is good but it'll be better to be consistent!

Reply:
consolidated on Optional. I was trying to stay away from the "Optional" keyword for these semantically mandatory arguments but yeah agreed that consistency is the key.

Reply:
agreed - consolidated on Optional